code/game/gamemodes/objective.dm

```diff
@@ -263,12 +307,12 @@
 						return 1
 	return 0
 
-/datum/objective/block
+/datum/objective/escape_obj/block

```

Comment:
I'll have to check over it but I'm pretty sure this doesn't count the user, so you can use this to escape with the AI's help. If this doesn't work the way I think it does/doesn't play well, I'll blacklist it